std/getopt.d

```diff
@@ -468,13 +467,13 @@ unittest
    string and its bound pointer.
 */
 enum config {
-    /// Turns case sensitivity on
+    /// Turn case sensitivity on
     caseSensitive,
-    /// Turns case sensitivity off
+    /// Turn case sensitivity off
     caseInsensitive,
-    /// Turns bundling on
+    /// Turn bundling on
     bundling,
-    /// Turns bundling off
+    /// Turn bundling off
```

Comment:
why did you remove the "s" if one views the docs.

caseSensitive\n Turns case sen.... 

looks ok